src/aap_eda/core/enums.py

```diff
@@ -89,6 +89,8 @@ class CredentialType(DjangoStrEnum):
     REGISTRY = "Container Registry"
     GITHUB = "GitHub Personal Access Token"
     GITLAB = "GitLab Personal Access Token"
+    EXTRA_VAR = "Extra Var"
```

Comment:
On a second thought the "encrypted" value could suggest that the value must be encrypted so it might be confusing, but I feel we should use the full name, something "extra variable" to be consistent with the rest of the options.

Reply:
Like what we did in the demo, a more descriptive title means we need more keyins to create it

Reply:
Since the UI is going to be displaying it the text won't matter like we did in swagger.
The Extra Var terminology is familiar to ansible users so we need that.Maybe Vaulted Extra Var